src/Guards/TokenGuard.php

```diff
@@ -170,15 +163,6 @@ protected function authenticateViaBearerToken($request)
             $psr->getAttribute('oauth_access_token_id')
         );
 
-        $clientId = $psr->getAttribute('oauth_client_id');
-
-        // Finally, we will verify if the client that issued this token is still valid and
-        // its tokens may still be used. If not, we will bail out since we don't want a
-        // user to be able to send access tokens for deleted or revoked applications.
-        if ($this->clients->revoked($clientId)) {
```

Comment:
Is this check still happening somewhere else?

Reply:
$this->clients->findActive() above makes sure the $client is not revoked.